frontend/app/tests/unit/components/settings/api-keys/service-key.spec.ts

```diff
@@ -1,17 +1,25 @@
 import { mount, Wrapper } from '@vue/test-utils';
 import { createPinia, PiniaVuePlugin, setActivePinia } from 'pinia';
 import Vue from 'vue';
+import { useI18n } from 'vue-i18n-composable';
 import Vuetify from 'vuetify';
 import ServiceKey from '@/components/settings/api-keys/ServiceKey.vue';
-import '../../../i18n';
+import { mockT, mockTc } from '../../../i18n';
 
 Vue.use(Vuetify);
 Vue.use(PiniaVuePlugin);
 
+vi.mock('vue-i18n-composable');
```

Comment:
Wouldn't it be better to use the factory here to mock useI18n? (not blocking btw, mostly for future consideration)